config/helm/appmesh-controller/values.yaml

```diff
@@ -122,6 +122,14 @@ stats:
 # Enable cert-manager
 enableCertManager: false
 
```

Comment:
Let's also [test.yaml](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/config/helm/appmesh-controller/test.yaml)

Also, you may want to increment the Charts version here: https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/config/helm/appmesh-controller/Chart.yaml#L4

Reply:
ACK